### ISSUE TITLE
shellcheck errors: missing array quoting

### ISSUE THREAD

**Issue by pabs3** [kokoye2007/wifi-qr#9]
There are two shellcheck errors:

```
$ shellcheck --severity error wifi-qr

In wifi-qr line 114:
  for l in ${wifilist[@]}; do
           ^------------^ SC2068: Double quote array expansions to avoid re-splitting elements.


In wifi-qr line 120:
  for m in ${array[@]}; do
           ^---------^ SC2068: Double quote array expansions to avoid re-splitting elements.
```

If you remove the severity flag you will see that there are lots of other issues with the script too.

**Comment by kokoye2007** [comment]
Thanks
I will fix and update at 0.1-2 version.
<3

**Comment by pabs3** [comment]
It seems you missed the other issues, please run this:

```
$ shellcheck wifi-qr
```

BTW, you can close issues from git commit logs like this:

   Fixes: https://github.com/kokoye2007/wifi-qr/issues/9

So there is no need to manually close the git issues.

-- 
bye,
pabs

https://wiki.debian.org/PaulWise
